app/src/main/java/com/alexchan/wallpaper/MainActivity.kt

```diff
@@ -1,12 +1,124 @@
 package com.alexchan.wallpaper
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.GravityCompat
+import androidx.navigation.fragment.NavHostFragment
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.android.material.navigation.NavigationView
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity(), View.OnClickListener {
 
-class MainActivity : AppCompatActivity() {
+//    private val REQUEST_CODE = 100
+
+    // For Log statement
+    companion object {
+        private val TAG: String?= MainActivity::class.simpleName
+    }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        // Set TopToolbar OnClickListener
+        topToolbar.setOnClickListener{openNavDrawer()}
+
+        // Set NavigationView Drawer OnItemCLickListener
+        navigation_view.setNavigationItemSelectedListener(drawerNavListener())
```

Comment:
Usually we will name the view with camel case, like navigationView, bottomNavigation.